common/DevHomeAdaptiveCards/CardModels/DevHomeChoicesData.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace DevHome.Common.DevHomeAdaptiveCards.CardModels;
+
+/// <summary>
+/// Represents a choice in an adaptive card choice set. This is the same as the adaptive card choices
+/// type in the adaptive card schema with an added subtitle property. This can be updated in the future
+/// to add other properties.
+/// </summary>
+public class DevHomeChoicesData
```

Comment:
Does this class need to stay in sync with adaptive card choices type?

Reply:
I added a comment to advise future dev to keep it in sync